core/client/src/main/java/alluxio/client/file/options/ListStatusOptions.java

```diff
@@ -50,14 +60,85 @@ public LoadMetadataType getLoadMetadataType() {
   }
 
   /**
-   * @param loadMetadataType the loadMetataType
+   * @param loadMetadataType the loadMetadataType
    * @return the updated options
    */
   public ListStatusOptions setLoadMetadataType(LoadMetadataType loadMetadataType) {
     mLoadMetadataType = loadMetadataType;
     return this;
   }
 
+  /**
+   * @return raw size option. It specifies whether to print raw sizes
+   */
+  public boolean ismRawSize() {
+    return mRawSize;
+  }
+
+  /**
+   * @param rawSize Raw Size Option
+   */
+  public void setmRawSize(boolean rawSize) {
+    mRawSize = rawSize;
+  }
+
+  /**
+   * @return the recursive option. It specifies whether the path has to be traversed recursively
+   */
+  public boolean ismRecursive() {
+    return mRecursive;
+  }
+
+  /**
+   * @param recursive Recursive option. It specifies whether to display directories recursively
+   */
+  public void setmRecursive(boolean recursive) {
+    mRecursive = recursive;
+  }
+
+  /**
+   * @return Force Load Metadata Option. It specifies whether to force loading files in the
+   *         directory.
+   */
+  public boolean ismForceLoadMetaData() {
+    return mForceLoadMetaData;
+  }
+
+  /**
+   * @param forceLoadMetaData Option to force loading files in the directory
```

Comment:
`@param forceLoadMetadata the force load metadata option to use. It specifies whether to force loading files in the directory.`